packages/zmarkdown/__tests__/__snapshots__/old-suite-latex.test.js.snap

```diff
@@ -666,7 +666,7 @@ Indented thrice\\\\ref{thrice}.
 
 
 
-Indented four\\\\ref{four} times.
+Indented [four] times.
```

Comment:
. (Do we need to escape this? Md input is simply `Indented [four] times.`, latex comes out unchanged as you see here.)

Reply:
We don't need to escape that, indeed. Square brackets are only problematic with commands (inside other square brackets).